t/python/t0010-job.py

```diff
@@ -161,6 +178,27 @@ def test_13_job_kvs(self):
             self.assertTrue(flux.kvs.exists(self.fh, job_kvs_dir.path))
             self.assertTrue(flux.kvs.isdir(self.fh, job_kvs_dir.path))
 
+    def test_14_job_cancel_invalid_args(self):
+        with self.assertRaises(ValueError):
+            job.kill(self.fh, "abc")
+        with self.assertRaises(ValueError):
+            job.cancel(self.fh, "abc")
+        with self.assertRaises(OSError):
+            job.kill(self.fh, 123)
+        with self.assertRaises(OSError):
+            job.cancel(self.fh, 123)
+
+    def test_15_job_cancel(self):
+        self.sleep_jobspec = JobspecV1.from_command(["sleep", "1000"])
+        jobid = job.submit(self.fh, self.sleep_jobspec, waitable=True)
+        job.cancel(self.fh, jobid)
+
+    def test_16_job_kill(self):
+        self.sleep_jobspec = JobspecV1.from_command(["sleep", "1000"])
+        jobid = job.submit(self.fh, self.sleep_jobspec, waitable=True)
+        job_event_wait(self.fh, jobid, "start")
+        job.kill(self.fh, jobid)
```

Comment:
Would it be worth it to toss a `job.wait(self.fh, jobid)` in here to make sure the job the completes (same for the cancel test)?  If we do that, maybe we want to also check that the `success` return of `wait()` is `False`?

Reply:
Great idea!